README.adoc

```diff
@@ -193,6 +193,19 @@ Preserve second fetch during initial checkout::
   This setting is only needed if there is a bug in the redundant fetch removal logic.
   If you enable this setting, please report a git plugin issue that describes why you needed to enable it.
 
+[[do-not-add-git-tag-action-to-jobs]]
+Do not add git tag action to jobs::
```

Comment:
could this be the default? I can't see people wanting to use this these days

Reply:
I worried that changing the default behavior to "disabled" might require a larger version increment and more explanation in release notes or elsewhere.  I've hesitated to change defaults of many things for fear of breaking existing use cases that were unknown to me but important to the user.

Reply:
if you don't change the default though will people actually discover it?

it should be a good out of the box experience for all users imo, I would just add to release notes users can opt out via configuration, and can always revert the default if people flood out not wanting this?

Reply:
Experiments yesterday confirmed that git tag action only works consistently if the workspace is on the Jenkins controller.  Workspaces on a Jenkins agent failed to apply the requested tag in all the cases I tested.

I'll switch the default to be disabled and allow administrators to enable it.  I'll use that as the excuse to call the release of this capability "git plugin 4.5.0".  I'll include some additional comments in the documentation about the old behavior and will rework the documentation and the image to match the new default of disabling that action.